README.md

```diff
@@ -114,35 +131,74 @@ Thus, using extended conformance via metadata, a dataset may:
+CLDF metadata uses terms from the [CLDF Ontology](http://cldf.clld.org/v1.0/terms.rdf), as specified in the file [`terms.rdf`](terms.rdf), 
+to mark 
+- [`TableGroup`](http://w3c.github.io/csvw/metadata/#table-groups) objects as representing a [CLDF module](modules),
+- [`Table`](http://w3c.github.io/csvw/metadata/#tables) objects as representing a [CLDF component](components) or
+- individual columns as representing [CLDF properties](http://cldf.clld.org/v1.0/terms.rdf#properties).
 
-The CLDF Ontology also provides a set of [properties](http://cldf.clld.org/v1.0/terms.rdf#properties) to attach 
-semantics to individual columns. While many of these properties are similar (or identical) to properties defined 
+While many of these properties are similar (or identical) to properties defined 
 elsewhere - most notably in the [General Ontology for Linguistic Description - GOLD](http://linguistics-ontology.org/) - 
 we opted for inclusion to avoid ambiguity, but made sure to reference the related properties in the ontology.
 
-Note that the column *names* in the default table descriptions (e.g. [`formTable`](components/forms)) are not always 
-the same as the column *properties*. Each column has both a `csvw:name` and a separate `propertyURL` linking the column 
-to the CLDF Ontology. Each property also has a `rdf:label` which might also be different.
+> [!NOTE]
+> The column *names* in the default table descriptions (e.g. for [`formTable`](components/forms)) are not always 
+> the same as the column *properties*. Each column has both a `csvw:name` and a separate `propertyURL` linking the column 
+> to the CLDF Ontology. Each property also has a `rdf:label` which might also be different.
 
-**Notes:**
-- In an ill-advised attempt to version the ontology, `v1.0` has been baked into the term
-  URIs. While this may be a good idea in case of incompatible changes (e.g. if the semantics
-  of a term changed), it presents an obstacle for initeroperability in case of backwards-compatible
-  changes. So starting with CLDF 1.1, we will keep `http://cldf.clld.org/v1.0/terms.rdf`
-  as namespace for **all** versions of the 1.x series, and specify the particular version
-  when a term was introduced using `dc:hasVersion` properties per term.
-- For better human readability the [CLDF Ontology](http://cldf.clld.org/v1.0/terms.rdf) should
-be visited with a browser capable of rendering XSLT - such as Firefox.
+> [!CAUTION]
+> In an ill-advised attempt to version the ontology, `v1.0` has been baked into the term
+> URIs. While this may be a good idea in case of incompatible changes (e.g. if the semantics
+> of a term changed), it presents an obstacle for interoperability in case of backwards-compatible
+> changes. So starting with CLDF 1.1, we will keep `http://cldf.clld.org/v1.0/terms.rdf`
+> as namespace for **all** versions of the 1.x series, and specify the particular version
+> when a term was introduced using `dc:hasVersion` properties per term.
```

Comment:
No real comment for changes, just wanted to mention that I think this is a very important comment (and helpful clarification) for people reading the specification, particularly when they encounter the first `dc:conformsTo` in the minimal example.